### DISCUSSION TITLE
Brute force in windows 10

### DISCUSSION THREAD

**Discussion by Pok4** [aircrack-ng/aircrack-ng#2370] [General]
Hello,
Im on win 10 and i have already last release.
How to start a brute force?
My laptop is PRIMUX IOXBOOK 1406F.
Can you give me how to guide, because im a noob.
I see that there is a gui exe, but i dont have a passwords list..
Please explain to me...

**Comment by Pok4** [comment]
The integrated wifi card is realtek 8822CE.

**Comment by Mister-X-** [comment]
Bruteforce is not realistic.

The best way to get started and learn is with the tutorials on our website.

**Reply by Pok4** [reply]
What can you suggest me? I look over the "Reaver"... What is the best option for me?
I can install Kali or Ubuntu, but i dont know from where to start...

I have 3 net boxes near me with wpa/wpa2 mixed mode and good signal..

What to do? Please give me some links and how to articles..

My netbook is cheap but have ddr4, new cpu and ssd. 2000 passwords for less than 1 minute..(I tested today)

**Comment by aircrack-ng** [comment]
Attacking AP that you're not authorized to attack is against the rules: https://github.com/aircrack-ng/aircrack-ng/discussions/2369

Thread closed.
